system-test/storage.ts

```diff
@@ -1263,6 +1263,28 @@ describe('storage', () => {
     });
   });
 
+  describe('bucket versioning', () => {
+    describe('bucket', () => {
+      it('should create a bucket with versioning enabled', async () => {
+        const bucket = storage.bucket(generateName());
+        await storage.createBucket(bucket.name, {
+          versioning: {
+            enabled: true,
+          },
+        });
+        await bucket.getMetadata();
+        assert.strictEqual(bucket.metadata.versioning.enabled, true);
+      });
+
+      it('should by default create a bucket without versioning set', async () => {
+        const bucket = storage.bucket(generateName());
+        await storage.createBucket(bucket.name);
+        await bucket.getMetadata();
+        assert.strictEqual(bucket.metadata.versioning.enabled, null);
```

Comment:
```suggestion
        assert.strictEqual(bucket.metadata.versioning, undefined);
```
I believe in the default case `metadata` won't have the `versioning` property.

Reply:
Ok, fixed this and repushed